ncm-sysctl/src/main/perl/sysctl.pm

```diff
@@ -22,23 +22,20 @@ use NCM::Component;
 our @ISA = qw(NCM::Component);
 use LC::Exception qw(throw_error);
 our $EC=LC::Exception::Context->new->will_store_all;
+our $NoActionSupported = 1;
 
+use CAF::Process;
 use EDG::WP4::CCM::Element;
-
 use NCM::Check;
-use LC::Process qw(trun);
-
-# For convenience
-my $base = '/software/components/sysctl';
 
 
 ##########################################################################
 sub Configure {
```

Comment:
remove the `####` lines, the `{` should start on a new line
(i know this is not your code :smile: )